lib/common/pool.c

```diff
@@ -159,14 +174,21 @@ void POOL_add(void* ctxVoid, POOL_function function, void *opaque) {
 
     pthread_mutex_lock(&ctx->queueMutex);
     {   POOL_job const job = {function, opaque};
-        /* Wait until there is space in the queue for the new job */
+
+        // Wait until there is space in the queue for the new job.
```

Comment:
You have to use C-style comments (`/* */`) in zstd code, because some compiles don't accept C++/C99 style comments (`//`).